<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiWidget</class>
 <widget class="QWidget" name="GuiWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GuiWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#TopWidget{background-color: rgb(176,196,222,255);}
#BodyWidget{background-color: rgb(192,192, 192,255);}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="20,80">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="TopWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_top1">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="label_theme">
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>320</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
color: rgba(255, 255, 255);
background-color:rgba(0,0,0,0);
font: 16pt &quot;Cantarell&quot;;
background-repeat: no-repeat;
outline: none;
}</string>
          </property>
          <property name="text">
           <string>人工智能识别系统</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_top2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButtonFirstPage">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>33</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>33</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{background-color:rgb(100,80,80,100);color:white;border-style: inset;}
QPushButton:hover{background-color:rgb(216,191,216);color:red;}
QPushButton:pressed{background-color:rgb(100,149,237);border-style: inset;}
QPushButton:focus{outline: none;}
</string>
            </property>
            <property name="text">
             <string>文件识别</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSecondPage">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>33</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>33</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{background-color:rgb(100,80,80,100);color:white;border-style: inset;}
QPushButton:hover{background-color:rgb(216,191,216);color:red;}
QPushButton:pressed{background-color:rgb(100,149,237);border-style: inset;}
QPushButton:focus{outline: none;}
</string>
            </property>
            <property name="text">
             <string>图片识别</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonThirdPage">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>33</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>33</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{background-color:rgb(100,80,80,100);color:white;border-style: inset;}
QPushButton:hover{background-color:rgb(216,191,216);color:red;}
QPushButton:pressed{background-color:rgb(100,149,237);border-style: inset;}
QPushButton:focus{outline: none;}
</string>
            </property>
            <property name="text">
             <string>邮件识别</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_top3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnMinimal">
            <property name="maximumSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{border-image: url(:/icon/config/img/min_btn.png);}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnMaximized">
            <property name="maximumSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{border-image: url(:/icon/config/img/max_btn.png);}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnClose">
            <property name="maximumSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{border-image: url(:/icon/config/img/close_btn.png);}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_top4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="BodyWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
